components/NewsfeedCard/NewsfeedCardBody.tsx

```diff
@@ -0,0 +1,176 @@
+import CardBootstrap from "react-bootstrap/Card"
+import { useMediaQuery } from "usehooks-ts"
+import { Col, Row } from "../bootstrap"
+
+interface NewsfeedCardBodyProps {
+  imgSrc?: string
+  imgAltTxt?: string
+  text?: string
+  timestamp?: string
+}
+
+export const NewsfeedBillCardBody = (props: NewsfeedCardBodyProps) => {
+  const { imgAltTxt, text, timestamp } = props
+  const isMobile = useMediaQuery("(max-width: 550px)")
+  return (
+    <CardBootstrap.Body className={`p-0`}>
+      <div
+        className={`align-items-center bg-info d-flex mx-2 mt-1 mb-2 p-2 rounded`}
+      >
+        {isMobile ? (
+          <>
+            <Col className={`d-flex align-items-center flex-column m-2`}>
+              <img
+                src="\images\clock.svg"
+                height="105.6"
+                width="105.6"
+                alt={imgAltTxt}
+                className={`m-3`}
+              />
+              <CardBootstrap.Text className={`mb-0 mt-2`}>
+                <strong>{text}</strong>
+              </CardBootstrap.Text>
+              <>
+                {"Action taken on "}
+                {timestamp}
+              </>
+            </Col>
+          </>
+        ) : (
+          <>
+            <img
+              src="\images\clock.svg"
+              height="105.6"
+              width="105.6"
+              alt={imgAltTxt}
+              className={`m-3`}
+            />
+            <Col className={`m-2`}>
+              <CardBootstrap.Text className={`mb-0`}>
+                <strong>{text}</strong>
+              </CardBootstrap.Text>
+              <>
+                {"Action taken on "}
+                {timestamp}
+              </>
+            </Col>
+          </>
+        )}
+      </div>
+    </CardBootstrap.Body>
+  )
+}
+
+export const NewsfeedTestimonyCardBody = (props: NewsfeedCardBodyProps) => {
+  const { imgSrc, imgAltTxt, text, timestamp } = props
+  const isMobile = useMediaQuery("(max-width: 550px)")
+
+  function capitalizeFirstLetter(x: String | undefined) {
+    return String(x).charAt(0).toUpperCase() + String(x).slice(1)
```

Comment:
nit: could this be a utility function we just import? Seems like this would be handy in other places